app/helpers/commodities_helper.rb

```diff
@@ -144,4 +144,14 @@ def declarable_heading_full(commodity)
         tag.p(commodity.to_s.html_safe)
     end
   end
+
+  def four_four_two_commodity_code(code)
```

Comment:
the name is a bit cryptic, what about 
`pretty_commodity_code` or `readable_commodity_code`?

Reply:
Yeh - naturally I'd call it something like `format_commodity_code` but we're already formatting it several different ways - hence trying to incorporate the format into the name.

How about `segmented_commodity_code` ?

Reply:
Renamed to `segmented_commodity_code`